src/test/lib/concurrency/stress_test.cpp

```diff
@@ -587,4 +587,105 @@ TEST_F(StressTest, OperatorRegistration) {
   }
 }
 
+// For cases where issues usually arise early, we run the test multiple times.
+class StressTestMultipleRuns : public StressTest, public ::testing::WithParamInterface<int> {};
```

Comment:
We don't actually use the parameter in the code, right? It's bascially just a loop around the test body. I guess it's a matter of taste, but I don't clearly see why we need a parameterized test class for that anymore (totally understand why it whas there for finding the loop count though).